test/tests/performance/pbs_sched_perf.py

```diff
@@ -455,6 +457,17 @@ def test_multi_sched_perf(self):
                 endtime = PBSLogUtils.convert_date_time(
                     log_msg[1].split(';')[0])
                 dur = endtime - start
-                self.perf_test_result(dur, sc + "_cycle_duration", "secs")
-                self.logger.info('Time taken by ' + str(sc) +
-                                 ' scheduler to run 1k jobs is ' + str(dur))
+                if i == 0:
+                    max = dur
+                else:
+                    if dur > max:
+                        max = dur
```

Comment:
For one, you don't need i.  dur can't be negative, so this if statement will work.  The other option is to create a list of durations, and then call max(list) at the end.

Reply:
i is just used as a flag to know that it is first attempt. I am just storing the max value of the durations.